assignments/mg-dts002/assign.diagnostics.json

```diff
@@ -0,0 +1,28 @@
+{
+    "properties": {
+      "metadata": {
+        "assignedBy": "PlatOps"
+      },
+      "description": "Policy to set diagnostic setting to forward Activity Logs to an Azure Event Hub",
+      "displayName": "HMCTS Diagnostic Policy",
+      "enforcementMode": "DoNotEnforce",
```

Comment:
why do not enforce?

Reply:
Was keeping it as that just as a placeholder. Will update.

Reply:
Updated